library/src/main/java/com/novoda/downloadmanager/notifications/DownloadNotifier.java

```diff
@@ -1,13 +1,21 @@
 package com.novoda.downloadmanager.notifications;
 
 import com.novoda.downloadmanager.lib.DownloadBatch;
+import com.novoda.downloadmanager.lib.NotificationsCreatedListener;
 
 import java.util.Collection;
 
 public interface DownloadNotifier {
+    int TYPE_ACTIVE = 1;
```

Comment:
They need to be public because they're used in two packages now. Hopefully will be able to move em back into package-private scope shortly